tests/test_compile_error.rs

```diff
@@ -62,4 +62,5 @@ fn test_compile_errors() {
     #[cfg(all(Py_LIMITED_API, not(Py_3_9)))]
     t.compile_fail("tests/ui/abi3_dict.rs");
     t.compile_fail("tests/ui/duplicate_pymodule_submodule.rs");
+    t.compile_fail("tests/ui/invalid_base_class.rs");
```

Comment:
Given the diagnostic contains extra detail on `abi3`:

```suggestion
    #[cfg(not(Py_LIMITED_API))]
    t.compile_fail("tests/ui/invalid_base_class.rs");
```